core/src/forest/ForestTrainers.cpp

```diff
@@ -62,6 +63,21 @@ ForestTrainer regression_trainer() {
                        std::move(prediction_strategy));
 }
 
+ForestTrainer local_linear_trainer(double split_lambda,
+                                   bool weight_penalty,
+                                   std::vector<double> overall_beta,
```

Comment:
These should be `const` references to avoid unnecessary copying, for example
```
const std::vector<double>& overall_beta
```

Reply:
Okay makes sense! I checked and for other LL methods, we don't do this for the vector of split variables; should I update those to const as well?